### PR TITLE
Add labels to the image

### PR DESCRIPTION
**Pull request by magthe**
The commit message attempts to explain the reason for this.

There's a build of Rooster made with this change: `zimpler/rooster:test-labels-for-metadata-1408`

``` shell
$ docker inspect zimpler/rooster:test-labels-for-metadata-1408 | jq '.[0].ContainerConfig.Labels'
{
  "com.zimpler.branch": "test-labels-for-metadata",
  "com.zimpler.build_id": "1408",
  "com.zimpler.commit": "072fbc54799026c58f607a63040358bad5e0c4e3",
  "com.zimpler.system": "rooster"
}
```

Do note that fd53df0459de9be0f2410da32d1d474a77d08804 only is present to be able to test it, i.e. do not merge it!

### CODE REVIEW

File: build-docker [hunk 1]
```diff
@@ -12,9 +12,10 @@ fi
 
 # Build image
 docker build -t ${APP_NAME}:latest \
-       --build-arg TRAVIS_BUILD_NUMBER=${TRAVIS_BUILD_NUMBER} \
-       --build-arg TRAVIS_COMMIT=${TRAVIS_COMMIT} \
-       --build-arg TRAVIS_BRANCH=${TRAVIS_BRANCH} \
+       --label "com.zimpler.system=${APP_NAME}" \
+       --label "com.zimpler.build_id=${TRAVIS_BUILD_NUMBER}" \
+       --label "com.zimpler.commit=${TRAVIS_COMMIT}" \
+       --label "com.zimpler.branch=${TRAVIS_BRANCH}" \
```
Comment (GaspardP) [inline build-docker]: Regarding the `/status` route, we now expect the information to be set during compilation (which will happen on travis). Correct?

File: setup_travis.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 pip install --user awscli # install aws cli w/o sudo
 export PATH=$PATH:$HOME/.local/bin # put aws in the path
 
-curl https://raw.githubusercontent.com/Zimpler/ecs-deploy-scripts/master/build-docker > "$HOME/.local/bin/build-docker"
+curl https://raw.githubusercontent.com/Zimpler/ecs-deploy-scripts/use-labels-for-metadata/build-docker > "$HOME/.local/bin/build-docker"
```
Comment (hsvallfors) [inline setup_travis.sh]: this should be master when this is merged, right?

Comment (GaspardP) [reply setup_travis.sh]: Yes, that's why the change is isolated in the `tmp! Point the setup script to this branch ` commit :+1:

Comment (hsvallfors) [reply setup_travis.sh]: Ok, I have no feedback then :+1:

File: build-docker (hunk 1 above)
Comment (Jell) [reply build-docker]: Ruby projects still need the build-arg for the status page

Comment (magthe) [reply build-docker]: @GaspardP Yes, for compiled code that's true.

@Jell Good catch!

**Comment by Jell** [comment]
I hope you were not waiting for me to approve @magthe I'm out of the reviewing loop I'm afraid :p (although I did leave a comment). Approving now in case you were waiting for it

**Pull request merged**